.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+
+Typos, errors, lack of examples and/or explanation, etc. This are some examples of for things that could be fixed and/or improved.
+
+You could even make improvements to this guide! :)
+
+While documenting, try to keep things simple and clear.
+
+### Issues
+
+Some issues are created with missing information, without a template, not reproducible, or plain
+invalid.
+
+You can make them easier to understand and resolve.
+
+#### Submitting an issue
+
+- Please search for similar issues before opening a new one;
+- Use one of the corresponding issue template;
+- Use a clear and descriptive title;
+- Include as much information as possible by filling out the provided issue
+  template;
+- A lot of times, the best way to report an issue is a failing test proving it.
+
+### Feedback
+
+The more feedback the better! We're always looking for more suggestions and opinions on discussions. That's a good opportunity to influence the future direction of this tool.
+
+This include submitting an enhancement suggestion, including completely new features and minor improvements to existing functionality.
+
+The
+[`question` label](https://github.com/Rocketseat/unform/labels/question)
+is a good place to find ongoing discussions.
+
+### Code
+
+You can use issue labels to discover issues you could help out with:
+
+- [`bug` issues](https://github.com/Rocketseat/unform/labels/bug)
+  are known bugs we'd like to fix;
+- [`enhancement` issues](https://github.com/Rocketseat/unform/labels/enhancement)
+  are features we're open to including.
+
+The
+[`help wanted`](https://github.com/Rocketseat/unform/labels/help%20wanted)
+and
+[`good first issue`](https://github.com/Rocketseat/unform/labels/good%20first%20issue)
+labels are especially useful.
+
+You may find an issue is assigned. Please double-check before starting on this issue because somebody else is likely already working on it.
+
+#### Dev environment
+
+When developing, prefer using **Node** ≥ 8 and **yarn**. Writing code with the latest stable Node versions allows us to use newer developer tools.
+
+After [cloning the repository](https://help.github.com/articles/cloning-a-repository/), run `yarn` to install dependencies.
+
+A summary of the scripts:
+
+- To run the example use `yarn dev:example`;
+- The `yarn dev:start` command will build the lib and watch files in bundle and rebuild on changes;
+- Use `yarn test` to run the test suite (powered by [Jest](https://facebook.github.io/jest/));
+- Run `yarn coveralls` to provide test coverage statistics;
+- `yarn build` will build the lib using [Rollup](https://rollupjs.org/guide/en);
+
+This project uses [Prettier](http://prettier.io/) for code formatting. Consider installing an [editor plugin](https://prettier.io/docs/en/editors.html) for the best experience, but code will also be formatted with a precommit script (using [lint-staged](https://github.com/okonet/lint-staged)).
```

Comment:
Prettier can be configured inside VSCode without the plugin itself, don't know if it's a important note.

Reply:
Nice. didn't knew about that. But I don't think it should be mentioned, since it's completely specific to VSCode.